webrtc.html

```diff
@@ -1932,26 +1935,29 @@
                   set and the set of certificates differs from the ones used
                   when <var>connection</var> was constructed, throw an
                   <code>InvalidModificationError</code>.</li>
-                  <li>
-                    <p>Let the value of
-                    <code><var>configuration</var>.<a data-for=
-                    "RTCConfiguration">iceTransportPolicy</a></code> be the
-                    <a>ICE Agent</a>'s <dfn id="ice-transports-setting">ICE
-                    transports setting</dfn>.</p>
+                  <li>If <code><var>configuration</var>.<a data-for=
+                  "RTCConfiguration">bundlePolicy</a></code> is
+                  set and its value differs from the <var>connection</var>'s
+                  bundle policy, throw an <code>InvalidModificationError</code>.
+                  </li>
+                  <li>If <code><var>configuration</var>.<a data-for=
+                  "RTCConfiguration">rtcpMuxPolicy</a></code> is
+                  set and its value differs from the <var>connection</var>'s
+                  rtcpMux policy, throw an <code>InvalidModificationError</code>.
                   </li>
                   <li>
-                    <p>Let the value of
+                    <p>Set the <a>ICE Agent</a>'s <dfn id="ice-transports-setting">
+                    ICE transports setting</dfn> to the value of
                     <code><var>configuration</var>.<a data-for=
-                    "RTCConfiguration">bundlePolicy</a></code> be
-                    <var>connection</var>'s bundle policy.</p>
+                    "RTCConfiguration">iceTransportPolicy</a></code>.</p>
```

Comment:
Yes, that is stated in JSEP Section 3.5.1, so we need to make that clear (as well as referencing that section of JSEP).